cfpq_data/graphs/utils/edges_statistics.py

```diff
@@ -0,0 +1,85 @@
+"""Returns an edge statistics."""
+import logging
+from typing import Dict, List, Any
+
+import networkx as nx
+
+__all__ = ["get_labels_frequency", "get_sorted_labels"]
+
+
+def get_labels_frequency(graph: nx.MultiDiGraph) -> Dict[Any, int]:
+    """Returns a dictionary with the number of edge labels used in the graph.
+
+    Parameters
+    ----------
+    graph : MultiDiGraph
+        Given graph.
+
+    Examples
+    --------
+    >>> from cfpq_data import *
+    >>> g = labeled_two_cycles_graph(1, 1, labels=("a", "b"))
+    >>> list(g.edges(data=True))
+    [(1, 0, {'label': 'a'}), (0, 1, {'label': 'a'}), (0, 2, {'label': 'b'}), (2, 0, {'label': 'b'})]
+    >>> labels_frequency = get_labels_frequency(g)
+    >>> labels_frequency
+    {'a': 2, 'b': 2}
+
+    Returns
+    -------
+    labels_frequency : Dict[Any, int]
+        Dictionary with edge labels usage frequency.
+    """
+    labels_frequency = dict()
```

Comment:
I think `defaultdict` would be better.
```suggestion
    labels_frequency = defaultdict(int)
```

Reply:
Yes, I agree for `nested_parentheses_grammar_from_graph` and `dyck_grammar_from_graph`. However, `java_points_to_grammar_from_graph` automatically finds set of fields used in java programs corresponding to our java graphs. For now, only web description of java graphs have information about which fields they have. You think `java_points_to_grammar_from_graph` should be also removed?

Reply:
defaultdict fixed.

Reply:
> Yes, I agree for `nested_parentheses_grammar_from_graph` and `dyck_grammar_from_graph`. However, `java_points_to_grammar_from_graph` automatically finds set of fields used in java programs corresponding to our java graphs. For now, only web description of java graphs have information about which fields they have. You think `java_points_to_grammar_from_graph` should be also removed?

Oh, now I see, the `java_points_to_grammar_from_graph` function is unambiguous. I think we should keep it.